app/templates/ui/app/search/search-results.html

```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="result in results">
   <h4>
-    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
+    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
```

Comment:
Since we don't provide a title by default I would just do .uri.

Unless we show them how to easily supply a title.